.travis.yml

```diff
@@ -3,7 +3,7 @@
 # therefore deactivated. Activating Travis for your own fork is as easy as
 # activating it in the travis site of your fork.
 
-sudo: required
+sudo: false
```

Comment:
I don't know if it's still true but at the time we also used that to use the new containerized infra.

Maybe they are all run on the same infra, now.

Reply:
I don't know about earlier, but currently, according to [this](https://docs.travis-ci.com/user/reference/overview/#Virtualisation-Environment-vs-Operating-System), `sudo: required` means we *don't* use containers, but VMs.